docs/index.html

```diff
@@ -1,15 +0,0 @@
-<head>
```

Comment:
I'd prefer to move this commit to another PR, as this one is already quite dense and this removal has no relationship with the PR itself.

Reply:
Yep.